### PR TITLE
Subwallet Feature updates

### PR DESCRIPTION
**Pull request by DrW3RK**

### CODE REVIEW

File: docs/build/build-wallets.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ developed wallets.
 | [Stylo](https://stylo-app.com/)               | [1](https://polkadot.polkassembly.io/treasury/39)                                                                                                                                                                                                     | Live              | Thibaut Sardan  | Android      | Air-gapped offline wallet             |
 | [Nova Wallet](https://novawallet.io/)\*       | [1](https://kusama.polkassembly.io/treasury/122), [2](https://kusama.polkassembly.io/treasury/158)                                                                                                                                                    | Live              | Nova Foundation | iOS, Android | Staking, Nomination Pools, Crowdloans, parachain accounts, Parity Signer and Ledger support            |
```
Comment (filippoweb3) [inline docs/build/build-wallets.md]:
```suggestion

| [Nova Wallet](https://novawallet.io/)\*       | [1](https://kusama.polkassembly.io/treasury/122), [2](https://kusama.polkassembly.io/treasury/158)                                                                                                                                                    | Live              | Nova Foundation | iOS, Android | Staking, Nomination Pools, Crowdloans, Governance, Parachain Accounts, Parity Signer and Ledger support            |

```

Comment (filippoweb3) [reply docs/build/build-wallets.md]: Nova supports Gov too

**Review by filippoweb3 — APPROVED** [review]
lgtm, minor addition about Nova.

**Comment by DrW3RK** [comment]
Looks like the formatting is a bit off. I will fix and merge it later @filippoweb3

**Pull request merged**